spec/search-api.yaml

```diff
@@ -175,38 +175,26 @@ components:
             or group of tables, then the source must be absent. An example of this would be an error occurring from a bug
             in the implementation. Additionally, if the error is due to a bad request from the client, the source must
             also be absent.
-        message:
+        status:
+          type: integer
+          format: int32
+          description: |
+            HTTP status code (as per [RFC 7231](https://tools.ietf.org/html/rfc7231)) generated by the server for this occurrence of the problem.
+            This must match the status code in the actual HTTP response. Used for convenience of clients (e.g. to determine what the original status code was in cases where it has been changed by an intermediary or cache or when message bodies persist without HTTP information).
```

Comment:
In this use case, we are allowing "partial success" in the results, and there can be more than one error returned in the `errors` array.

I don't think we should require the HTTP response to match the status code in this object.